packages/threat-composer/src/components/workspaces/WorkspaceInsights/components/Overview/index.tsx

```diff
@@ -20,13 +20,14 @@ import ColumnLayout from '@cloudscape-design/components/column-layout';
 import { LinkProps } from '@cloudscape-design/components/link';
 import { css } from '@emotion/react';
 import { useMemo, FC, ReactNode } from 'react';
-import { LEVEL_HIGH, LEVEL_LOW, LEVEL_MEDIUM, LEVEL_NOT_SET } from '../../../../../configs';
-import { useAssumptionLinksContext } from '../../../../../contexts';
+import { LEVEL_HIGH, LEVEL_LOW, LEVEL_MEDIUM, LEVEL_NOT_SET, MITIGATION_STATUS_IDENTIFIED, MITIGATION_STATUS_IN_PROGRESS, MITIGATION_STATUS_RESOLVED, MITIGATION_STATUS_RESOLVED_ABANDONED, STATUS_NOT_SET, THREAT_STATUS_IDENTIFIED, THREAT_STATUS_NOT_USEFUL, THREAT_STATUS_RESOLVED } from '../../../../../configs';
```

Comment:
Upon reflection, 'Abandoned' carries a negative connotation - and that it is simply left behind or neglected. I propose we change to 'Will not action' which carries slightly less negative connotation in my opinion, and is clear that it's a _decision_:

`MITIGATION_STATUS_RESOLVED_WILLNOTACTION`